todo_app/app.py

```diff
@@ -10,51 +10,45 @@ def create_app():
     @app.route('/')
     def index():
 
-        trello_list_ids = {'todo':get_trello_list_id('To Do'),'doing':get_trello_list_id('Doing'),'done':get_trello_list_id('Done')}
+        list_ids = {'todo':'todo','doing':'doing','done':'done'}
 
-        items = get_trello_cards()
+        items = get_todo_cards()
 
-        item_view_model = ViewModel(items, trello_list_ids)
+        item_view_model = ViewModel(items, list_ids)
 
         todays_date = datetime.datetime.strftime(datetime.date.today(), '%d/%m/%Y')
 
         if request.values.get('sort') == '1':
-            items.sort(key=lambda x: x.idList)
+            items.sort(key=lambda x: x.due_date)
         elif request.values.get('sort') == '2':
-            items.sort(key=lambda x: x.idList, reverse=True)
+            items.sort(key=lambda x: x.due_date, reverse=True)
+        else:
+            items.sort(key=lambda x: x.due_date)
```

Comment:
Again minor, but when two branches lead to the same outcome it's often easy to combine some of this logic, e.g.:
```python
        if request.values.get('sort') == '2':
            items.sort(key=lambda x: x.due_date, reverse=True)
        else:
            items.sort(key=lambda x: x.due_date)
```